### PR TITLE
feat(NavBar)!: renames Navbar to NavBar; remove forced composition

### PR DESCRIPTION
**Pull request by slingner**
Related to issue #74
- Updated the casing of NavBar
- Moved withStyles outside of scope so it is not re-rendering
- Updated SvgIcon's connected to the NavBar story
- Fixed fontsize to be rem
- Took out forced Monogram and toolbar, added to story instead

** Left NavbarLink alone and will address in a future PR

### CODE REVIEW

**Review by seigler — APPROVED** [review]
I don't know what's happening with Chromatic CI, but the code looks good.

**Comment by WilliamKelley** [comment]
Looks like the Chromatic CI is failing on the lab components. You updated that package in this PR. When you run lovally, do those stories work?

You might have to revert those changes.

**Review by WilliamKelley — CHANGES_REQUESTED** [review]
see above

**Comment by slingner** [comment]
> see above

Yes, my storie

> see above

Reverted back to .58, that seemed to fix the Chromatic issues. Thanks, William.

File: libs/spark/src/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  root: {
+    borderBottom: `2px solid ${palette.grey.medium}`,
+    fontSize: '1.125rem',
+    color: palette.text.onLightLowContrast,
+  },
+})(AppBar);
+
+export const NavBar = ({ color = 'default', ...other }: NavBarProps) => {
+  return (
+    <StyledAppBar position="fixed" elevation={0} color={color} {...other} />
```
Comment (WilliamKelley) [inline libs/spark/src/NavBar.tsx:19]: I think we should also set `component="nav"`, the [landmark HTML element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav).

**Review by WilliamKelley — CHANGES_REQUESTED** [review]

File: libs/spark/src/NavBar.tsx (hunk 1 above)
Comment (slingner) [reply libs/spark/src/NavBar.tsx]: Added ✅

**Review by WilliamKelley — APPROVED** [review]
Looks good. I approved the Chromatic changeset as well

**Comment by slingner** [comment]
> Looks good. I approved the Chromatic changeset as well

Apologies if I missed this, Is that something I should do next time on my PR's?

**Comment by WilliamKelley** [comment]
> 
> 
> > Looks good. I approved the Chromatic changeset as well
> 
> Apologies if I missed this, Is that something I should do next time on my PR's?

Ideally a reviewer should do it. But if everyone approved the PR and no-one approves the Chromatic changeset, you can assume implicit approval and approve it yourself in Chromatic.

**Comment by slingner** [comment]
> > > Looks good. I approved the Chromatic changeset as well
> > 
> > 
> > Apologies if I missed this, Is that something I should do next time on my PR's?
> 
> Ideally a reviewer should do it. But if everyone approved the PR and no-one approves the Chromatic changeset, you can assume implicit approval and approve it yourself in Chromatic.
> 
> Sorry this isn't outlined in CONTRIBUTING, it should be, just haven't gotten around to it.

No problem! You have been doing an amazing job with all of these Spark updates 🚀

**Pull request merged**